main.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background: #b7ffae;
+}
+
+.form-style-10 {
+  max-width: 450px;
+  padding: 30px;
+  margin: 40px auto;
+  background: #fff;
+  border-radius: 10px;
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.13);
+  -moz-box-shadow: 0 0 10px rgba(0, 0, 0, 0.13);
+  -webkit-box-shadow: 0 0 10px rgba(0, 0, 0, 0.13);
+}
+
+.form-style-10 .inner-wrap {
+  padding: 30px;
+  background: #f8f8f8;
+  border-radius: 6px;
+  margin-bottom: 15px;
+}
+
+.title {
+  background: #2aad56;
+  padding: 20px 30px 15px 30px;
+  margin: -30px -30px 30px -30px;
+  border-radius: 10px 10px 0 0;
+  -webkit-border-radius: 10px 10px 0 0;
+  -moz-border-radius: 10px 10px 0 0;
+  color: #fff;
+  text-shadow: 1px 1px 3px rgba(0, 0, 0, 0.12);
+  font: normal 30px "Bitter", serif;
+  -moz-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  -webkit-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  border: 1px solid #257c9e;
+}
+
+.form-style-10 h1 > span {
+  display: block;
+  margin-top: 2px;
+  font: 13px Arial, Helvetica, sans-serif;
+}
+
+.form-style-10 label {
+  display: block;
+  font: 13px Arial, Helvetica, sans-serif;
+  color: #888;
+  margin-bottom: 15px;
+}
+
+.form-style-10 input[type="text"],
+.form-style-10 input[type="date"],
+.form-style-10 input[type="datetime"],
+.form-style-10 input[type="email"],
+.form-style-10 input[type="number"],
+.form-style-10 input[type="search"],
+.form-style-10 input[type="time"],
+.form-style-10 input[type="url"],
+.form-style-10 input[type="password"],
+.form-style-10 textarea,
+.form-style-10 select {
```

Comment:
- [ ] CSS can get very messy if we don't use CSS class selectors. It's easier to identify and pinpoint an element on an HTML file this way. 

```
BAD
.form-style-10 textarea

GOOD
.textarea
```